### ISSUE TITLE
TODO: Color qL SOP (? -- discuss)

### ISSUE THREAD

**Issue by johnnyquest** [qLab/qLib#477]
Should work the same way as the regular Color SOP, but should also be able to
- specify color attribute name
- composite over existing color
- apply gamma

**Comment by JacobSanta** [comment]
Does the Attrib ramp not already have most of this functionality?

**Comment by johnnyquest** [comment]
@JacobSanta yeah -- i was thinking more along the lines of a regular Color SOP replacement (although it's true that you can pull it off with an Attrib Ramp, but still it would be a bit awkward...)
